OpenRA.Mods.RA/Activities/Infiltrate.cs

```diff
@@ -56,5 +58,13 @@ protected override void OnInside(Actor self)
 				Game.Sound.PlayNotification(self.World.Map.Rules, self.Owner, "Speech",
 					notification, self.Owner.Faction.InternalName);
 		}
+
+		public override Activity Tick(Actor self)
+		{
+		if (self.Trait<Infiltrates>().IsTraitDisabled)
+			return NextActivity;
```

Comment:
I don't think this is a valid thing to do.  `Enter` uses a child activity to move the actor, and I expect that bypassing that part way through will break things.

The correct thing to do in this case is to cancel the local activity.